packages/jest-runner/src/runTest.ts

```diff
@@ -139,6 +145,8 @@ async function runTestInternal(
     : null;
 
   const cacheFS = {[path]: testSource};
+
+  // @ts-ignore
   setGlobal(environment.global, 'console', testConsole);
```

Comment:
This too, something to do with typing of `Environment.$JestEnvironment`

Reply:
corrected it